docs/config/README.md

```diff
@@ -2933,7 +2934,7 @@ set to a number and meets or exceeds the specified threshold.
 | ----------- | ---------------------------- | ------------------------------------------------------------- |
 | `threshold` | `2`                          | Display threshold.                                            |
 | `format`    | `"[$symbol$shlvl]($style) "` | The format for the module.                                    |
-| `symbol`    | `"↕️  "`                     | The symbol used to represent the `SHLVL`.                     |
+| `symbol`    | `"↕️"`                       | The symbol used to represent the `SHLVL`.                     |
```

Comment:
Not sure what happened here, but this seems to have gone in the wrong direction since I reviewed. The symbol needs to have two spaces after it before the close quote (how it was in the original documentation).